src/bacnet/basic/npdu/h_npdu.c

```diff
@@ -43,6 +43,7 @@
 static uint16_t Local_Network_Number;
 static uint8_t Local_Network_Number_Status = NETWORK_NUMBER_LEARNED;
 
+extern uint8_t priority;  /* Fixing test 10.1.2 Network priority */
```

Comment:
Please use the same pattern as Local_Network_Number (naming, static scope) and a 'getter' function so as to not use a global variable 'priority'.